yolo/utils/metrics.py

```diff
@@ -138,3 +142,16 @@ def compute_ap(recall, precision):
         ap = np.sum((mrec[i + 1] - mrec[i]) * mpre[i + 1])  # area under curve
 
     return ap, mpre, mrec
+
+def compute_eer_pr(recalls,precisions):
+    """ Computes the equal error rate precision and recall from the precision and recall curves.
+    # Arguments
+        recall:    Recall curve (n sized list of recalls along the recall curve, evaluated at equal intervals)
+        precision: Precision curve (n sized list of recalls along the recall curve, evaluated at equal intervals)
+    # Returns
+        The equal error rate precison, recall, and thershold. Returns NaN if equal error rate could not be found (shouldn't happen with an actual classifier)
+    """
+    for i, (precision,recall) in enumerate(zip(precisions,recalls)):
+        if recall >= precision:
+            return recall,precision,i/recalls.shape[0]
```

Comment:
This isn't returning the actual threshold. I think you need to pass the thresholds in as a 3rd function argument and return the one that triggers the change.

Reply:
The thresholds in the conf array don't quite map to the right threshold so some kind of mapping is needed. I found a bit cleaner way to do it though